tools/odbc/CMakeLists.txt

```diff
@@ -54,6 +54,6 @@ add_library(
 set_target_properties(duckdb_odbc PROPERTIES DEFINE_SYMBOL "DUCKDB_ODBC_API")
 target_link_libraries(duckdb_odbc ${LINK_LIB_LIST} duckdb_static)
 
-if(NOT WIN32 AND NOT CLANG_TIDY)
+if(NOT CLANG_TIDY)
```

Comment:
Does this mean that we want to test ODBC for 32-bit Windows? From my understanding, otherwise, it only runs the tests for 64-bit, is that correct?

Reply:
I'm asking because supporting Windows 32-bit with ODBC might cause some potentially tricky issues in the future, and I am wondering if it's worth it.

Reply:
[For legacy reasons WIN32 just means Windows](https://stackoverflow.com/questions/61776207/where-does-win32-come-from-when-im-using-windows-64bit)

Reply:
Oh, super interesting, thanks!